python/MyriaPythonWorker.py

```diff
@@ -123,6 +124,9 @@ def write_with_length(cls, obj, stream, output_type):
         elif output_type == DataType.BLOB:
             cls.write_int(DataType.BLOB, stream)
             cls.pickle_and_write(obj, stream)
+        elif output_type ==DataType.EXCEPTION:
+            cls.write_int(len(obj), stream)
```

Comment:
BTW, if the object is always expected to be the output of `traceback.format_exc()`, then it is a `unicode` object, so we should assert `type(obj) is unicode` and just call `obj.encode("utf-8")`.